pages/integrations/slack.md.erb

```diff
@@ -30,21 +30,7 @@ Once created, the Slack channel and workspace cannot be changed. To post to a di
 
 By default, notifications are sent to all configured Slack channels.
 
-To restrict which Slack channels receive notifications:
-
-1. Add a [`notify`](/docs/pipelines/notifications#conditional-notifications) YAML block with a conditional `if` attribute to your `pipeline.yml` 
-
-	```yaml
-	notify:
-	  - slack: buildkite-community#general
-	    if: build.state == "passed"
-	```
-
-1. Update your Buildkite Slack notification service's 'Pipeline' setting to 'Only Some Pipelines', and un-check the pipeline with the conditional `notify` block
-
-	<%= image "buildkite-slack-notification.png", width: 1470/2, height: 1620/2, alt: "Screenshot of Add Buildkite Slack Notification" %>
-
-Further information about adding `notify` to your pipeline can be found in the [Notifications](/docs/pipelines/notifications#conditional-notifications) guide.
+For extra control over which channels receive notifications, it's possible to [define notification rules within pipeline YAML files](/docs/pipelines/notifications#conditional-notifications).
```

Comment:
What’s the rationale for removing this section entirely? Should we provide a hint about turning the automatic notifications off if you’re using the YAML version?

Reply:
What do people think about a second paragraph in the intro along the lines of 

- you can do x y and z in the UI
- to do 1 and 2 you need to use the YAML